src/components/area-chart.js

```diff
@@ -0,0 +1,182 @@
+/* Area chart */
+/* import line and material-ui */
+import React, { Component } from 'react'
+import { Line } from 'react-chartjs-2'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+/* creating a class for Areachart */
+class AreaChart extends Component {
+  constructor (props) {
+    super(props)
+
+    this.options = {
+      animation: {
+        duration: 2000
+      },
+      responsive: true,
+      tooltips: {
+        mode: 'label'
+      },
+      elements: {
+        line: {
+          fill: false
+        }
+      },
+
+      scales: {
+        xAxes: [
+          {
+            display: true,
+            gridLines: {
+              display: false
+            },
+            labels: {
+              show: true
+            },
+            labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July']
+          }
+        ],
+        yAxes: [
+          {
+            type: 'linear',
+            display: true,
+            position: 'left',
+            id: 'y-axis-1',
+            gridLines: {
+              display: false
+            },
+            labels: {
+              show: true
+            }
+          },
+          {
+            type: 'linear',
+            display: true,
+            position: 'right',
+            id: 'y-axis-2',
+            gridLines: {
+              display: false
+            },
+            labels: {
+              show: true
+            }
+          }
+        ]
+      }
+    }
+    /* set random data to chart */
+    this.state = {
+      data: {
+        datasets: [{
+          label: 'my first dataset',
```

Comment:
please change the 'my first data set' to something more fancy and professional. Lets say 'productivity', 'revenue in CAD' and 'Improved efficiency'